### PR TITLE
Adding cart page

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Pages/Cart/CartComponents/AmountCard/AmountCard.jsx
- [ ] src/Pages/Cart/CartComponents/CartCard/CartCard.jsx
- [ ] src/Pages/Cart/CartComponents/CartCardContainer/CartCardContainer.jsx
- [ ] src/context/Cart.context.js

## What not to check
- [ ] Do Not Check Any Other Files

Link to live site,
https://stormpoint-shop.netlify.app/

### CODE REVIEW

File: src/Pages/Cart/CartComponents/AmountCard/AmountCard.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    setTotal(
+      cartProducts.reduce((acc, curr) => acc + Number(curr.price) * curr.qty, 0)
+    );
+  }, [cartProducts]);
+
+  return (
+    <div className="card-container card-shadow">
+      <h2 className="card-subtitle bold">Price Detail</h2>
+      <div className="filter-divider-line"></div>
+      <h2 className="card-subtitle">
```
Comment (hsnice16) [inline src/Pages/Cart/CartComponents/AmountCard/AmountCard.jsx:20]: Here, `<h2>` are repeating so you can use an array here to map.

You can make an array of objects.

File: src/Pages/Cart/CartComponents/CartCard/CartCard.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  const { loading, dispatch } = useFilterReducerContext();
+  const { cartProducts, removeFromCart, changeCartQty } = useCartContext();
+
+  return (
+    <>
+      {loading ? (
+        <Loader />
+      ) : (
+        <>
+          {cartProducts.map((products) => (
```
Comment (hsnice16) [inline src/Pages/Cart/CartComponents/CartCard/CartCard.jsx:16]: You can remove, `{` and `<>` here. As, I have suggested in the last PR.

**Review by hsnice16 — APPROVED** [review]
Overall Code looks good. Nice work. Just added a few suggestions.

File: src/Pages/Cart/CartComponents/CartCard/CartCard.jsx (hunk 2 above)
Comment (b0llu) [reply src/Pages/Cart/CartComponents/CartCard/CartCard.jsx:16]: Yes its been taken care of for some reason its not in this PR i will fix it

File: src/Pages/Cart/CartComponents/AmountCard/AmountCard.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/Cart/CartComponents/AmountCard/AmountCard.jsx:20]: sure will think on this

**Review by anik31 — APPROVED** [review]
Amazing, works perfectly. Code is also good.

**Pull request merged**